### PR TITLE
added simple controller for pod management using XQueueJob definition

### PR DESCRIPTION
**Pull request by scostache**
Pod management using XQueueJob controller and template definition

### CODE REVIEW

**Comment by dmatch01** [comment]
@k82cn Hi Da, could you do a review and merge if possible?  Thanks!

**Comment by k82cn** [comment]
please add test, or append your UT result.

**Comment by scostache** [comment]
added test in e2e.go

File: test/e2e.go [hunk 1]
```diff
@@ -30,7 +30,14 @@ var _ = Describe("E2E Test", func() {
 		err := waitJobReady(context, queueJob.Name)
 		Expect(err).NotTo(HaveOccurred())
 	})
-
+	It("Schedule XQueueJob", func() {
+		context := initTestContext()
+		defer cleanupTestContext(context)
+		rep := clusterSize(context, oneCPU)
+		xqueueJob := createXQueueJob(context, "xqj-1", 2, rep, "busybox", oneCPU)
```
Comment (k82cn) [inline test/e2e.go:38]: Where defined this helper function?

Comment (k82cn) [reply test/e2e.go:38]: Right now, we did not have e2e in CI; it need to make sure e2e passed in your local environment.

**Comment by scostache** [comment]
@k82cn  : test passed: 
• [SLOW TEST:10.215 seconds]
E2E Test
/root/gopath/src/github.com/kubernetes-incubator/kube-arbitrator/test/e2e.go:24
  Schedule QueueJob
  /root/gopath/src/github.com/kubernetes-incubator/kube-arbitrator/test/e2e.go:25
------------------------------
• [SLOW TEST:10.805 seconds]
E2E Test
/root/gopath/src/github.com/kubernetes-incubator/kube-arbitrator/test/e2e.go:24
  Schedule XQueueJob
  /root/gopath/src/github.com/kubernetes-incubator/kube-arbitrator/test/e2e.go:34
------------------------------

**Pull request closed without merging**